api/lib/block-listeners/cosmos-node-subscription.js

```diff
@@ -51,6 +53,25 @@ class CosmosNodeSubscription {
     }, PROPOSAL_POLLING_INTERVAL)
   }
 
+  async updateValidators() {
```

Comment:
I don't really like the solution as it is only Cosmos. Can we have a solution that works for all networks?

Reply:
You can have the same logic of querying the database to get the popularity in the store. The store is used by every source the same way.

Reply:
yes, sorry :facepalm:  At some point I forgot about Polkadot